managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -0,0 +1,464 @@
+
+The import procedure depends on your data source:
+    - [Importing data from an existing PostgreSQL database](#importing-data-from-an-existing-postgresql-database)
+      - [Using pg_dump](#using-pgdump)
+      - [Using pgAdmin](#using-pgadmin)
+      - [Using Airbyte](#using-airbyte)
+      - [Using Fivetran](#using-fivetran)
+      - [Using Meltano](#using-meltano)
+    - [Importing data from an existing PostgreSQL backup](#importing-data-from-an-existing-postgresql-backup)
+    - [Importing data from a file (.csv, .txt, etc.)](#importing-data-from-a-file-csv-txt-etc)
+
+## Importing data from an existing PostgreSQL database
+
+### Using pg_dump
+
+#### Prerequisites
+
+To complete this procedure, you must have installed PostgreSQL 16 (or newer) with [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) and [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
+
+#### Downloading and importing data into a Managed Database for PostgreSQL
+
+1. Run the following command to download a local export of your database with `pg_dump`, then enter your password when prompted:
+
+    ```
+    pg_dump --no-privileges --no-owner -U {username} -h {host} --port {port} -Fc {databasename} > my-backup
+    ```
+
+    <Message type="tip">
+    You can download specific tables using the `-t` option:
+
+      ```
+      pg_dump -t table1name -t table2name --no-privileges --no-owner -U {username} -h {host} --port {port} -Fc {databasename} > my-backup
+      ```
+    </Message>
+
+2. When finished, make sure the backup is downloaded on your local machine. The default location is in the current directory, under `/my-backup`.
+
+3. Run the command below to import data into your Managed Database for PostgreSQL using `pg_restore`. Make sure to replace the placeholders with your Managed Database for PostgreSQL connection parameters:
+
+    ```
+    pg_restore --no-privileges --no-owner --clean --if-exists -U {username} -h {host} --port 5432 -d {databasename} my-backup
+    ```
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database.
+    </Message>
+
+4. Enter the database password you defined upon [Database Instance creation](/managed-databases/postgresql-and-mysql/how-to/create-a-database/) when prompted. 
+
+5. When finished, make sure your data is stored in your new database by [connecting to it](/managed-databases/postgresql-and-mysql/how-to/connect-database-instance/), and performing a query.
+
+### Using pgAdmin
+
+#### Prerequisites
+
+To complete the following procedure, you must have:
+
+- Installed [pg_Admin](https://www.pgadmin.org/).
+
+- Ensured your PGAdmin version is compatible with the PostgreSQL version of the database you will import your data into. Currently, pgAdmin 7.8 or newer is required to fully work with PostgreSQL 16.
```

Comment:
For Managed PostgreSQL, we need to remove "Currently, pgAdmin 7.8 or newer is required to fully work with PostgreSQL 16." and replace it with something like "Check this documentation https://www.pgadmin.org/docs/pgadmin4/7.8/release_notes.html to ensure compatibility"